### PR TITLE
Basic tests

### PR DESCRIPTION
**Pull request by muan**
Passes! https://travis-ci.org/github/image-crop-element/builds/396614492

- [ ] change repo description url once this is merged

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+describe('image-crop', function() {
+  describe('element creation', function() {
+    it('creates from document.createElement', function() {
+      const el = document.createElement('image-crop')
+      assert.equal('IMAGE-CROP', el.nodeName)
```
Comment (dgraham) [inline test/test.js]: Maybe also `assert(el instanceof window.ImageCropElement)`.

File: test/test.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          <input type="text" data-image-crop-input="height" name="height" aria-label="height">
+        </image-crop>
+      `
+    })
+
+    afterEach(function() {
+      document.body.innerHTML = ''
+    })
+
+    it('change event fires and updates input', function(done) {
```
Comment (dgraham) [inline test/test.js]: This would read more naturally as `it('fires a change event…')`.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+language: node_js
+sudo: required
```
Comment (dgraham) [inline .travis.yml]: I wonder if this workaround for Chrome security sandboxing is still needed. We might first try pushing without this setting to see if it works.

Comment (keithamus) [reply .travis.yml]: I was setting up Travis yesterday and ran into this; I'm guessing it is still needed.

Comment (muan) [reply .travis.yml]: Yeah it's needed. https://travis-ci.org/github/image-crop-element/builds/396955361

**Pull request merged**